### DISCUSSION TITLE
OAuth2 provider name

### DISCUSSION THREAD

**Discussion by AntonioSCoelho** [tolgee/tolgee-platform#1957] [Q&A]
Hi,

Is it possible to change the login button title for [OAuth2 authentication](https://tolgee.io/platform/self_hosting/configuration#oauth2)?

I'm trying to set up the Microsoft identity platform, and it works great, but the login button title is "OAuth2 login"

```
tolgee.authentication.oauth2.client-id=MY_ID
tolgee.authentication.oauth2.client-secret=MY_SECRET
tolgee.authentication.oauth2.scopes=openid email profile
tolgee.authentication.oauth2.authorization-url=https://login.microsoftonline.com/common/oauth2/v2.0/authorize
tolgee.authentication.oauth2.token-url=https://login.microsoftonline.com/common/oauth2/v2.0/token
tolgee.authentication.oauth2.user-url=https://graph.microsoft.com/oidc/userinfo
```

Something like this:
`tolgee.authentication.oauth2.name=Microsoft`

**Comment by JanCizmar** [comment]
Hello, this is part of feature SSO login feature, which we plan to release as a part of enterprise version. For free self-hosted version you'll have to stay with this default name.
